app/models/concerns/patient_reportable.rb

```diff
@@ -12,15 +12,15 @@ module PatientReportable
     scope :ltfu_as_of, ->(date) do
       joins("LEFT OUTER JOIN latest_blood_pressures_per_patient_per_months
              ON patients.id = latest_blood_pressures_per_patient_per_months.patient_id
-             AND #{sanitize_sql(["bp_recorded_at > ? AND bp_recorded_at < ?", date - LTFU_TIME, date])}")
-        .where("bp_recorded_at IS NULL AND patients.recorded_at < ?", date - LTFU_TIME)
+             AND #{sanitize_sql(["bp_recorded_at > ? AND bp_recorded_at < ?", (date - LTFU_TIME).end_of_month, date])}")
+        .where("bp_recorded_at IS NULL AND patients.recorded_at < ?", (date - LTFU_TIME).end_of_month)
```

Comment:
This is slightly unrelated: This fixes the problem we saw in February where `12.months.ago` is not the right date to display.

Reply:
Wow, nice one. This Feb, it would have shown Feb 28 2020, even though the end of that month was Feb 29 (leap year)? Amazing ✨

Reply:
Yep 😅